modules/extensions/galasa-extensions-parent/dev.galasa.auth.couchdb/src/main/java/dev/galasa/auth/couchdb/internal/CouchdbAuthStore.java

```diff
@@ -206,12 +206,13 @@ public List<IUser> getAllUsers() throws AuthStoreException {
     @Override
     public void createUser(String loginId, String clientName, String roleId) throws AuthStoreException {
 
+        logger.info("createUser: Creating user loginId"+loginId+" clientName:"+clientName+" roleId:"+roleId );
```

Comment:
Colon missing here in the logger message might make things hard to read if the loginId starts with letters potentially

Reply:
fixed